README.md

```diff
@@ -143,11 +143,21 @@ If you want to have a sound played when the character reaches your tile, your im
 
 ```javascript
     visit: function() {
```

Comment:
It's not needed, I guess. Just mention the name of the file. Licenses don't need code :+1: